src/database/appConfig/appConfig.module.ts

```diff
@@ -1,10 +1,10 @@
 import { Module } from '@nestjs/common';
 import { AppConfigService } from './appConfig.service';
-import { AppConfig } from './appConfig.entity';
+import { Config } from './config.entity';
 import { TypeOrmModule } from '@nestjs/typeorm';
 
 @Module({
-  imports: [TypeOrmModule.forFeature([AppConfig])],
+  imports: [TypeOrmModule.forFeature([Config])],
   providers: [AppConfigService],
   exports: [AppConfigService],
 })
```

Comment:
I avoided renaming `AppConfigModule` to `ConfigModule` due to a naming conflict with the built-in NestJS `ConfigModule`. Similarly, `AppConfigService` was kept unchanged to prevent issues in `app.module.ts`.